components/test_sst_importer/src/lib.rs

```diff
@@ -4,18 +4,30 @@ use std::fs;
 use std::path::Path;
 
 use crc::crc32::{self, Hasher32};
+use engine_rocks::Rocks;
+use engine_traits::KvEngine;
 use kvproto::import_sstpb::*;
 use uuid::Uuid;
 
-use engine::rocks::{SstWriterBuilder, DB};
+use engine::rocks::util::new_engine;
```

Comment:
I'm not clear on the role of the `engine` crate, though it looks to me like it is doing a similar thing as the engine_traits/engine_rocks crates, but incomplete. I don't have a problem depending on components/engine with the expectation that it will eventually disappear.